src/views/app.tsx

```diff
@@ -106,4 +37,21 @@ const AppRouting = () => (
   </tokenManager.Auth0Provider>
 );
 
+function ViewRedirect() {
```

Comment:
This is great! I think we should redirect on `/?id={recordingId}` too since the current version on `app.replay.io` produces that URL.